src/olympia/addons/api_urls.py

```diff
@@ -11,7 +11,7 @@
 
 
 addons = SimpleRouter()
-addons.register(r'addon', AddonViewSet)
+addons.register(r'addon', AddonViewSet, base_name='addon')
```

Comment:
DRF was using `queryset` to find the model name and therefore the default `base_name`.